### ISSUE TITLE
tnil

### ISSUE THREAD

**Issue by Colonizor48** [melopee/ithkey#2]
Does this support TNIL?(New ithkuil)

**Comment by 0-claire** [comment]
No. Tho I'm about to upload a TNIL keyboard today or so

**Comment by 0-claire** [comment]
No, though I've made an x11 (for Linux running Xorg) keyboard for TNIL. I'll upload it soon
